admin/load.php

```diff
@@ -15,13 +15,8 @@
  * @since 1.0.0
  */
 function perflab_add_modules_page() {
```

Comment:
Do we need to change the functions name as well?

Reply:
+1

Maybe just call it `perflab_add_settings_page()`, so it's not tied to modules/plugins internals.

Reply:
Addressed in https://github.com/WordPress/performance/pull/1060/commits/66f0492278e40c1c5a31790b9d20515977e6d3b3